Library/Homebrew/dev-cmd/boneyard-formula-pr.rb

```diff
@@ -70,6 +73,9 @@ def boneyard_formula_pr
       end
     end
     branch = "#{formula.name}-boneyard"
+
+    reason = " because #{reason}" if reason
```

Comment:
Punted just because it's unlikely it'll be more than you and me using this anyway 😀